scripts/install-nix-from-closure.sh

```diff
@@ -1,181 +1,700 @@
 #!/bin/sh
+### v 2.02-1-dev
+###
+###############################
+###
+### Nix installation script
+### Universal shell script for for OS Linux environments
+###
+###############################
+###
+### Upstream URL:
+### https://github.com/NixOS/nix/blob/master/scripts/install-nix-from-closure.sh
+###
+### Script is fully POSIX compatible
+### (POSIX standard at http://pubs.opengroup.org/onlinepubs/9699919799/)
+### In some places sript can have controlled put of POSIX moments.
+###
+### Script strives to be fully transactional, as much as shell script can be.
```

Comment:
Remove `, as much`..`can be`
There are no limitations to shell scripts in this context.